src/crimson/os/seastore/random_block_manager/rbm_device.h

```diff
@@ -342,67 +303,37 @@ class NVMeBlockDevice : public Device {
     ceph::bufferlist bl,
     uint16_t stream = 0) = 0;
 
-  /*
-   * For passsing through nvme IO or Admin command to SSD
-   * Caller can construct and execute its own nvme command
-   */
-  virtual nvme_command_ertr::future<int> pass_through_io(
-    nvme_io_command_t& command) { return seastar::make_ready_future<int>(0); }
-  virtual nvme_command_ertr::future<int> pass_admin(
-    nvme_admin_command_t& command) { return seastar::make_ready_future<int>(0); }
-
-  /*
-   * End-to-End Data Protection
-   *
-   * NVMe device keeps track of data integrity similar with checksum. Client can
-   * offload checksuming to NVMe device to reduce its CPU utilization. If data
-   * protection is enabled, checksum is calculated on every write and used to
-   * verify data on every read.
-   */
-   bool is_data_protection_enabled() const { return data_protection_enabled; }
-
-  /*
-   * Data Health
-   *
-   * Returns list of LBAs which have almost corrupted data. Data of the LBAs
-   * will be corrupted very soon. Caller can overwrite, unmap or refresh data to
-   * protect data
-   */
-   virtual nvme_command_ertr::future<std::list<uint64_t>> get_data_health() {
-     std::list<uint64_t> fragile_lbas;
-     return nvme_command_ertr::future<std::list<uint64_t>>(
-	nvme_command_ertr::ready_future_marker{},
-	fragile_lbas
-     );
-   }
-
-  /*
-   * Recovery Level
-   *
-   * Regulate magnitude of SSD-internal data recovery. Caller can get good read
-   * latency with lower magnitude.
-   */
-   virtual nvme_command_ertr::future<> set_data_recovery_level(
-     uint32_t level) { return nvme_command_ertr::now(); }
-
-  /*
-   * Predictable Latency
-   *
-   * NVMe device can guarantee IO latency within pre-defined time window. This
-   * functionality will be analyzed soon.
-   */
+  bool is_data_protection_enabled() const { return false; }
 };
 
 /*
  * Implementation of NVMeBlockDevice with POSIX APIs
  *
- * PosixNVMeDevice provides NVMe SSD interfaces through POSIX APIs which is
+ * NVMeBlockDevice provides NVMe SSD interfaces through POSIX APIs which is
  * generally available at most operating environment.
  */
-class PosixNVMeDevice : public NVMeBlockDevice {
```

Comment:
Commit title is wrong:
...
2. PosixNVMeDevice to NVMeBlockDevice

Reply:
-class PosixNVMeDevice : public NVMeBlockDevice {
+class NVMeBlockDevice : public RBMDevice {

PosixNVMeDevice was removed, then renamed NVMeBlockDevice.

Reply:
I meant the commit message is wrong:

> 2. rename PosixNVMeBlockDevice to NVMeBlockDevice

Should be `2. PosixNVMeDevice to NVMeBlockDevice`